test/e2e/self_hosted_test.go

```diff
@@ -24,16 +24,28 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var _ = Describe("When testing Cluster API working on self-hosted clusters", func() {
-
+var _ = Describe("When testing Cluster API working on single control plane self-hosted clusters", func() {
 	SelfHostedSpec(context.TODO(), func() SelfHostedSpecInput {
```

Comment:
Lets use the global contest introduced by https://github.com/kubernetes-sigs/cluster-api/pull/4706 instead of context.TODO